src/libs/ReportUtils.ts

```diff
@@ -774,6 +779,13 @@ function isReportApproved(reportOrID: OnyxInputOrEntry<Report> | string | EmptyO
     return report?.stateNum === CONST.REPORT.STATE_NUM.APPROVED && report?.statusNum === CONST.REPORT.STATUS_NUM.APPROVED;
 }
 
+/**
+ * Checks if the supplied report has been manually reimbursed
+ */
+function isReportManuallyReimbursed(report: OnyxEntry<Report>): boolean {
```

Comment:
Just curious, should we remove `Report` from the function name, since it's obvious that this relates to a report?

Reply:
debatable. I think having report in the func name makes it more readable. Especially if we're dealing with report and reportActions in the same file. It's easy to get confused when you're new. 

ReportUtils has a few functions that actually deal with a `reportAction` 😅 

<img width="1150" alt="image" src="https://github.com/Expensify/App/assets/29673073/bd930c75-6fb3-464a-9b01-69c218a1a6de">

Reply:
Haha yikes, ok let's keep it for now, we can rename later for consistency iffffff more people care :D